src/main/java/frc/robot/RobotContainer.java

```diff
@@ -84,18 +84,20 @@ private void configureButtonBindings() {
         shelfPosition.onTrue(new ArmCommand(Constants.ArmConstants.shelfPosition, false));
         zeroEncoder.onTrue(new ZeroArmCommand());
 
-        //When a button is pressed runs its respective method inside drawer
+        // When a button is pressed runs its respective method inside drawer.
         drawerExtender.onTrue(new DrawerCommand(true));
         drawerRetractor.onTrue(new DrawerCommand(false));
 
-        //When a button is pressed runs its respective method inside gripperCommand
+        // When a button is pressed runs its respective method inside gripperCommand.
         gripperOpen.onTrue(new GripperCommand(true));
         gripperClose.onTrue(new GripperCommand(false));
 
     }
 
+    /* 
     public Command getAutonomousCommand() {
-        return new AutoPath(s_Swerve);
+        return new Auto4();
```

Comment:
Why is auto 4 being returned here?

Reply:
Just leftover from testing that I missed removal of. It isn't needed anymore as everything is being done in robot.java.